src/component/ChairList/ChairList.js

```diff
@@ -4,29 +4,31 @@ import styled from 'styled-components';
 
 import ChairCard from '../ChairCard/ChairCard';
 
-export default function ChairList({chairsData, toggleDescription, showDescription}) {
+export default function ChairList({chairs, toggleDescription, showDescription, setFavoriteChair, toggleLike}) {
   const {name} = useParams();
   let navigate = useNavigate();
```

Comment:
```suggestion
  const navigate = useNavigate();
```
I know in the react-router-docs the example is with `let` however, `const` should be preferred :)